crates/meroctl/src/cli/context/create.rs

```diff
@@ -260,7 +260,9 @@ async fn update_context_application(
         &format!("admin-api/dev/contexts/{context_id}/application"),
     )?;
 
-    let request = UpdateContextApplicationRequest::new(application_id);
+    //How to make conversion from libp2p to public key
+    let public_key = keypair.public();
+    let request = UpdateContextApplicationRequest::new(application_id, public_key);
```

Comment:
no, it shouldn't be the libp2p keypair, this should be the `member_public_key` from the create context response

this was wrong about `context call` as well, which I [fixed some days ago](https://github.com/calimero-network/core/pull/948/files#diff-f5d45fa7e6c0662f1afc369ed781cd633d630781f205ce957203e02437621722L77-R97)